src/lib/core/tests/TestCHIPTLV.cpp

```diff
@@ -2983,7 +2983,7 @@ void TestCHIPTLVReaderDup(nlTestSuite * inSuite)
 void TestCHIPTLVReaderErrorHandling(nlTestSuite * inSuite)
 {
     CHIP_ERROR err;
-    uint8_t buf[2048];
+    uint8_t buf[2048] = { 0 };
```

Comment:
Why this change?

Reply:
There was a separate discussion that this buffer is not initialize and may actually include some valid TLV components.

Reply:
Presumably because the old code is broken and asserts things about the contents of un-initialized memory....  But it should probably not happen in this PR.